ui/app/utils/default_jobs/actions.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export default `job "redis-actions" {
+  datacenters = ["*"]
+
+  group "cache" {
+    count = 1
+    network {
+      port "db" {}
+    }
+
+    task "redis" {
+      driver = "docker"
+
+      config {
+        image = "redis:3.2"
+        ports = ["db"]
+        command = "/bin/sh"
+        args = ["-c", "redis-server --port \${NOMAD_PORT_db} & /local/db_log.sh"]
+      }
+
+      resources {
+        cpu = 500
+        memory = 256
+      }
```

Comment:
If possible, I would try to tighten up these values as it can result in placement failures in small deployments with other jobs running. I think redis can run with fewer resources.